app/services/tagging/common_ancestor_finder.rb

```diff
@@ -0,0 +1,54 @@
+module Tagging
+  class CommonAncestorFinder
+    def self.call
+      new.find_all
+    end
+
+    def find_all
+      content_enum = Services.rummager.search_enum({ fields: %w[content_id title] }, page_size: 100)
```

Comment:
We could use the list of in scope document types here, as that would avoid considering the tags to the ~70,000 HMRC Manual Sections, plus other stuff?

Reply:
Fixed - had to copy the BLACKLIST array, because the export_data* files are deprecated and will be removed in future

Reply:
How about using this blacklist https://github.com/alphagov/content-tagger/blob/master/config/document_types_excluded_from_the_topic_taxonomy.yml ? That is still relevant for the metrics, so won't be removed any time soon. It's also a better list, as it excludes less document types.

Reply:
ok, used those instead